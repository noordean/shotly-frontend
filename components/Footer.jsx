import React from 'react';

const Footer = () => (
  <div className="footer">
    Â© 2018 Copyright noordean.
  </div>
);

export default Footer;
