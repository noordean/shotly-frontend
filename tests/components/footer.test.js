import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../../components/Footer.jsx';

describe('<Footer />', () => {
  it('should render the necessary elements', () => {
      const rendered = renderer.create(
        <Footer />
      );
      expect(rendered.toJSON()).toMatchSnapshot();
  });
});
