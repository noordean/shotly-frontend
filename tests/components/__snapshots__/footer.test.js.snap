// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render the necessary elements 1`] = `
<div
  className="footer"
>
  Â© 2018 Copyright noordean.
</div>
`;
